source/Calamari/Kubernetes/KubernetesContextScriptWrapper.cs

```diff
@@ -31,10 +31,13 @@ public KubernetesContextScriptWrapper(CalamariVariableDictionary variables)
         /// <summary>
         /// One of these fields must be present for a k8s step
         /// </summary>
-        public bool IsEnabled(ScriptSyntax syntax) => (!string.IsNullOrEmpty(variables.Get(SpecialVariables.ClusterUrl, "")) ||
-                               !string.IsNullOrEmpty(variables.Get(SpecialVariables.AksClusterName, "")) ||
-                               !string.IsNullOrEmpty(variables.Get(SpecialVariables.EksClusterName, ""))) &&
-                                supportedScriptSyntax.Contains(syntax);
+        public bool IsEnabled(ScriptSyntax syntax)
+        {
+            return (!string.IsNullOrEmpty(variables.Get(SpecialVariables.ClusterUrl, "")) ||
+                    !string.IsNullOrEmpty(variables.Get(SpecialVariables.AksClusterName, "")) ||
+                    !string.IsNullOrEmpty(variables.Get(SpecialVariables.EksClusterName, ""))) &&
+                   supportedScriptSyntax.Contains(ScriptSyntaxHelper.GetPreferredScriptSyntaxForEnvironment());
```

Comment:
This should just be `syntax == ScriptSyntaxHelper.GetPreferredScriptSyntaxForEnvironment()`, and the `supportedScriptSyntax` field can be entirely removed.